### ISSUE TITLE
Cannot read "required" tooltip with VoiceOver

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1772]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Collection or component page, e.g. https://findingaids.princeton.edu/catalog/C0178

### Description of error / warning

When navigating with VoiceOver through the Ask a Question or Suggest a Correction modal forms, it doesn't tell me that the fields are required, it just announces the tooltip as "star" as in  "name star. message star."

### WCAG success criterion

[1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG22/#info-and-relationships)

"Semantic markup is appropriately used to designate headings, regions/landmarks, lists, emphasized or special text, etc."

[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG22/#name-role-value)

"ARIA is used appropriately to enhance accessibility when HTML is not sufficient."

### WCAG conformance level

A

### Recommended fix

Either use a "required" indicator that doesn't need a tooltip (this might be best) or add aria directives so that it's read correctly and the tooltip isn't needed.

### Tool used to find the problem

VoiceOver in Safari

### Screenshot

<img width="407" height="177" alt="Image" src="https://github.com/user-attachments/assets/b34e0365-2363-4198-b1af-e6e9577daf44" />
